src/net.cpp

```diff
@@ -1019,6 +1021,9 @@ void CConnman::CreateNodeFromAcceptedSocket(std::unique_ptr<Sock>&& sock,
     }
 
     const bool inbound_onion = std::find(m_onion_binds.begin(), m_onion_binds.end(), addr_bind) != m_onion_binds.end();
+    CNodeOptions node_opts;
+    node_opts.prefer_evict = discouraged;
+    node_opts.permission_flags = permissionFlags;
```

Comment:
It could be a bit shorter if designated initializers are used, instead of:

```cpp
Opts opts;
opts.x = 1;
opts.y = 2;
opts.z = 3;
new CNode(
    ...
    opts
);
```

it would become:

```cpp
new CNode(
    ...
    {
        .x = 1,
        .y = 2,
        .z = 3
    }
);
```

Reply:
Err, yeah; I had the same thought, but apparently didn't push it.